### DISCUSSION TITLE
How to specify a directory

### DISCUSSION THREAD

**Discussion by MangMax** [nonzzz/vite-plugin-compression#62] [Q&A] [answered]
How to specify the directory for compression, can it only be dist?

**Comment by nonzzz** [comment, marked as answer]
Plugin will automatically get your output directory. In fact only set `build.outDir` in your vite's config. 

https://github.com/nonzzz/vite-plugin-compression/blob/b7d4f6d1303c0a549db0c7127c2bc51c889ae210/src/index.ts#L36-L59


```ts


import { defineConfig } from 'vite'

export default defineConfig({
  // ... your config
  build: {
    outDir: 'target'
  }
})


```
